### ISSUE TITLE
Any chance for linux support

### ISSUE THREAD

**Issue by marvin1099** [AshuraStrike/plugin-manager-OBS#1]
Is there any chance for Linux support,
all the plugin updaters for obs plugins are for windows.
therefore support for Linux is missing so it would be very nice.

**Comment by AshuraStrike** [comment]
Yes, but probably not for this project, currently this project uses Windows Forms for it's UI, and it only works on Windows

I'm slowly working on remaking this project to use multiplatform UI elements, it's a work in progress, with no date for release
